### PR TITLE
Add isLicensePlate and isLuhnNumber interface definitions to @ types/validator

### PR DESCRIPTION
**Pull request by xiaoxizi2303**
Add isLicensePlate and isLuhnNumber interface definitions to @ types/validator

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/validator/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 3b5fae49d04b8fae75c662fa9119fa2424167db9
</p>

File: types/validator/index.d.ts [hunk 1]
```diff
@@ -808,6 +808,16 @@ declare namespace validator {
      */
     export function isLength(str: string, options?: IsLengthOptions): boolean;
 
+    /**
+     * check if the string matches the format of a country's license plate.
+     */
+    export function isLicensePlate(str: string, locale:string): boolean;
+
+    /**
+     * check if the string passes the Luhn algorithm check.
```
Comment (codershiba) [inline types/validator/index.d.ts:817]:
```suggestion
     * Check if the string passes the [Luhn algorithm check](https://en.m.wikipedia.org/wiki/Luhn_algorithm).
```

File: types/validator/index.d.ts [hunk 2]
```diff
@@ -808,6 +808,16 @@ declare namespace validator {
      */
     export function isLength(str: string, options?: IsLengthOptions): boolean;
 
+    /**
+     * check if the string matches the format of a country's license plate.
+     */
+    export function isLicensePlate(str: string, locale:string): boolean;
```
Comment (codershiba) [inline types/validator/index.d.ts:814]:
```suggestion
    export function isLicensePlate(str: string, locale: 'cs-CZ' | 'de-DE' | 'de-LI' | 'en-IN' | 'es-AR' | 'hu-HU' | 'pt-BR' | 'pt-PT' | 'sq-AL' | 'sv-SE' | 'any'): boolean;
```

**Review by ghost — CHANGES_REQUESTED** [review]
Please address comments

File: types/validator/index.d.ts (hunk 1 above)
Comment (xiaoxizi2303) [reply types/validator/index.d.ts]: Modified

File: types/validator/index.d.ts [hunk 3]
```diff
@@ -808,6 +808,16 @@ declare namespace validator {
      */
     export function isLength(str: string, options?: IsLengthOptions): boolean;
 
+    /**
+     * check if the string matches the format of a country's license plate.
```
[the remark this answers is not in the document]
Comment (xiaoxizi2303) [reply types/validator/index.d.ts]: Modified

File: types/validator/index.d.ts (hunk 2 above)
Comment (xiaoxizi2303) [reply types/validator/index.d.ts]: Modified

**Review by ghost — APPROVED** [review]

**Review by jakebailey — APPROVED** [review]

**Comment by xiaoxizi2303** [comment]
Hi, when will this pr be incorporated?

**Comment by jakebailey** [comment]
You can merge the PR yourself by following the instructions above.

**Comment by xiaoxizi2303** [comment]
Ready to merge

**Pull request merged**